docs/SpaceTemplates.rst

```diff
@@ -0,0 +1,229 @@
+                            <field name="appTitle"><string>Space Settings</string></field>
+                            <field name="removable"><boolean>false</boolean></field>
+                            <field name="order"><int>2</int></field>
+                            <field name="uri"><string>settings</string></field>
+                          </object>
+                        </value>
+                        ...
+                      </collection>
+                    </field>
+                  </object>
+                </object-param>
+              </init-params>
+            </component-plugin>
+        </external-component-plugins>
+
+In which:
+
+-  **Name**: ``Space Template Configuration``
+
+-  **Set-method**: ``registerSpaceTemplatePlugin``
+
+-  **Type**:
+   ``org.exoplatform.social.core.space.SpaceTemplateConfigPlugin``
+
+-  **Init-params**:
+
++----------------------+-------------------------------------------------------+
+| Object-param         | Description                                           |
++======================+=======================================================+
+| **template**         | Sets the **SpaceTemplate** to be defined.             |
++----------------------+-------------------------------------------------------+
+
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| Field name              | Possible value               | Description                                                                |
++=========================+==============================+============================================================================+
+| **name**                | ``string``                   | The name of the space template.                                            |
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| **visibility**          | ``string``                   | The visibility setting of the space.                                       |
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| **registration**        | ``string``                   | The registration setting for the space.                                    |
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| **bannerPath**          | ``string``                   | The path of the space banner file inside your extension war.               |
++-------------------------+------------------------------+----------------------------------------------------------------------------+
+| **homePageApplication** | ``SpaceApplication``         | The **Application** to use in the home page of a space.                    |
```

Comment:
IMO ` The **Application** ` shouldn't be bold. it should be in normal weight as seen in "applications" field definition

Reply:
ok